nixnet/types.py

```diff
@@ -388,7 +391,115 @@ def from_raw(cls, frame):
             constants.FrameType.CANFD_DATA: CanFrame,
             constants.FrameType.CANFDBRS_DATA: CanFrame,
             constants.FrameType.CAN_REMOTE: CanFrame,
+            constants.FrameType.LIN_DATA: LinFrame,
+            constants.FrameType.LIN_BUS_ERROR: LinFrame,
+            constants.FrameType.LIN_NO_RESPONSE: LinFrame,
```

Comment:
Where is `LIN_NO_RESPONSE` documented?  Trying to decide if it makes sense to map it to a `LinFrame`